com.github.IsmaelMartinez.teams_for_linux.yml

```diff
@@ -14,17 +14,14 @@ finish-args:
   - --share=network
   - --device=all
   - --filesystem=home
-  - --filesystem=xdg-run/dconf
-  - --filesystem=~/.config/dconf:ro
   - --filesystem=xdg-run/pipewire-0
-  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
-  - --talk-name=ca.desrt.dconf
   - --talk-name=org.kde.StatusNotifierWatcher
   - --talk-name=org.freedesktop.Notifications
   - --talk-name=org.freedesktop.PowerManagement
   - --talk-name=org.freedesktop.ScreenSaver
   - --talk-name=org.gtk.Notifications
   - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.freedesktop.portal.Background
```

Comment:
This is not needed, flatpak grants wildcard access to `org.freedesktop.portal.*` just run it with --log-session-bus and you'll see in the top

Reply:
I see, but there was a request to do so here: https://github.com/IsmaelMartinez/teams-for-linux/issues/1032#issuecomment-1871962083

Reply:
they are wrong unfortunately. Flatseal does not use this permission and the background toggle in Flatseal is not for this.

The background toggle updates the permission store and sets background to yes. That means you allowed it to run in background.